<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SharesWidget</class>
 <widget class="QWidget" name="SharesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="actionLayout">
     <item>
      <widget class="QLabel" name="actionLabel">
       <property name="text">
        <string>Action:</string>
       </property>
       <property name="buddy">
        <cstring>actionComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="actionComboBox">
       <property name="currentText">
        <string>Create</string>
       </property>
       <item>
        <property name="text">
         <string>Create</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Update</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Delete</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="shareLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="shareNameLabel">
       <property name="text">
        <string>Share name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="shareNameLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="folderPathLabel">
       <property name="text">
        <string>Folder path:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="folderPathLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="commentLabel">
       <property name="text">
        <string>Comment:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="commentLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="shareToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="folderToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="actionModifiersLabel">
     <property name="font">
      <font>
       <family>Sans Serif</family>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Action
Modifiers:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="modifiersLayout">
     <item>
      <widget class="QCheckBox" name="updateAllRegularShares">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Update all regular shares</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="updateAllHiddenShares">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Update all hidden non-administrative
shares</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="updateAllAdministrativeDrives">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Update all administrative
drive-letter shares</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="userLimitLabel">
     <property name="font">
      <font>
       <family>Sans Serif</family>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>User limit:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QVBoxLayout" name="userLayout">
     <item>
      <widget class="QRadioButton" name="noChangeUsers">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>No change</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="maximumAllowedUsers">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Maximum allowed</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="allowThisNumberOfUsers">
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Alow this number of users:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="numberOfUsers"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="accessBasedLabel">
     <property name="font">
      <font>
       <family>Sans Serif</family>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Access-based
Enumeration:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QVBoxLayout" name="accessLayout">
     <item>
      <widget class="QRadioButton" name="noChangeAccess">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>No change</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="enableAccess">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="disableAccess">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Disable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
